src/main/java/io/vavr/control/Try.java

```diff
@@ -1132,6 +1132,21 @@ public final Either<Throwable, T> toEither() {
         }
     }
 
+    /**
+     * Converts this {@code Try} to an {@link Either}, converting the Throwable (if present)
```

Comment:
I'm proposing this change for the documentation so that it is more precise in meaning, because one could legitimately use a `Try` value to wrap a `Throwable` instance as a success value.
```suggestion
     * Converts this {@code Try} to an {@link Either}, converting the Throwable (in the failure case)
```

Reply:
I had the same impression that the wording was strange, but I kept this to stay consistent with `toValidation(Throwable => U)`.

I would make this doc change to both of these methods then, since your point applies to both cases.